dojo/tools/bugcrowd_api/parser.py

```diff
@@ -118,27 +129,31 @@ def convert_endpoint(self, url):
         if result:
             return Endpoint.from_uri(result.group(0))
         else:
-            return Endpoint.from_uri('')
+            return Endpoint.from_uri("")
 
     def include_finding(self, entry):
         """Determine whether this finding should be imported to DefectDojo"""
-
+        # Valid states from the Bugcrowd API
         # "new" "out-of-scope" "not-applicable" "not-reproducible" "triaged" "unresolved" "resolved" "informational"
+
         allowed_states = [
             "new",  # Finding from a previous pentest
-            "out_of_scope",     # Fix for finding is being verified
-            "not_applicable",     # Finding is a duplicate within the pentest
-            "not_reproducible",       # Finding is found to be a false positive
-            "triaged",      # Finding is verified and valid
-            "unresolved",           # The finding is not yet verified by the pentest team
+            "out_of_scope",  # Fix for finding is being verified
+            "not_applicable",  # Finding is a duplicate within the pentest
+            "not_reproducible",  # Finding is found to be a false positive
+            "triaged",  # Finding is verified and valid
+            "unresolved",  # The finding is not yet verified by the pentest team
             "resolved",  # Finding is out of the scope of the pentest
-            "informational",      # The finding is not yet verified by the pentest team
+            "informational",  # The finding is not yet verified by the pentest team
         ]
 
         if entry["attributes"]["state"] in allowed_states:
             return True
         else:
-            print(entry["attributes"]["state"] + 'NOT IN ALLOWED STATES !')
+            print(
+                entry["attributes"]["state"]
```

Comment:
We need to remove every `print()`

Reply:
Not my prints 🥲 but seriously though, how do parsers report errors to users ? I wasn't sure how I can raise an exception that bubbles back to the UI

Reply:
there is a need to add more data back to the importer a feature is missing IMO. for now just raise a `ValueError` or use `LOGGER.debug()`.
Print is a no-go